pkg/agent/cniserver/server_windows.go

```diff
@@ -36,3 +36,9 @@ func updateResultDNSConfig(result *current.Result, cniConfig *CNIConfig) {
 	}
 	klog.Infof("Got runtime DNS configuration: %v", result.DNS)
 }
+
+// getInfraContainer returns the K8S_POD_INFRA_CONTAINER_ID field of CNIConfig on Windows, and the containerID is the
+// actual container ID. kubelet-windows sends CNIAdd request for all containers in the same Pod.
+func (c *CNIConfig) getInfraContainer() string {
```

Comment:
maybe `getInfraContainerID` would be a better name?

Reply:
Sure, updated.